src/DGtal/geometry/curves/ArithmeticalDSLKernel.h

```diff
@@ -191,7 +210,8 @@ namespace DGtal
     typedef SpaceND<2, TCoordinate> Space;
     typedef typename Space::Vector Vector;
     typedef std::pair<Vector, Vector> Steps;
-
+    typedef std::pair<int, int> Octant;
+    
     // ----------------------- static members ---------------------------------
   public:
     static const unsigned short ForegroundAdjacency = 4;  //adjacency
```

Comment:
for static const declarations, there may be compiler issues. Please use BOOST_STATIC_CONST macro to init the member.

http://www.boost.org/doc/libs/1_31_0/more/int_const_guidelines.htm